bugbug/repository.py

```diff
@@ -128,6 +132,8 @@ def _hg_log(revs):
             bug=rev[4],
             backedoutby=rev[5],
             author_email=rev[6],
+            files=rev[7],
+            file_copies=rev[8],
```

Comment:
Parse them here directly (decode and split).
file_copies can be a map (from original path to copied path).